shotgun_api3/shotgun.py

```diff
@@ -427,6 +427,7 @@ def __init__(self, sg):
         self.session_token = None
         self.authorization = None
         self.no_ssl_validation = False
+        self.localized = False
```

Comment:
i am not seeing any docs or comments to describe what this is. Please add at least a comment - ideally sphinx docs in context of the code, but it's also ok if these are separate and part of the rst file.